### PR TITLE
Update changelog (0.1.2)

### PR DESCRIPTION
**Pull request by gavanderhoorn**
Preparation for the `0.1.2` release.

Might want to include a fix for #170 if we can get it in.

### CODE REVIEW

**Comment by ted-miller** [comment]
> Might want to include a fix for https://github.com/Yaskawa-Global/motoros2/issues/170 if we can get it in.

I definitely want to get #170 in this. We have an active user experiencing this issue.

**Comment by gavanderhoorn** [comment]
> We have an active user experiencing this issue.

you mean besides me .. ? :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,23 @@ SPDX-License-Identifier: CC-BY-SA-4.0
...
+
+- Goal validation now includes a check for duplicated joint names ([#162](https://github.com/Yaskawa-Global/motoros2/pull/162))
+- INFORM job validator now prints offending lines to the debug log ([#167](https://github.com/Yaskawa-Global/motoros2/pull/167))
+
+Changes:
+
+- Lan port MAC addresses containing 0s no longer cause initialisation failures ([#145](https://github.com/Yaskawa-Global/motoros2/pull/145))
+- Alarm & subcode documentation was updated to include missing subcodes ([#151](https://github.com/Yaskawa-Global/motoros2/pull/151))
+- Generated INFORM jobs no longer fail validation after reboot on YRC1000micro ([#168](https://github.com/Yaskawa-Global/motoros2/pull/168))
+- `/tf` data now publishes correctly when using a base-axis ([#172](https://github.com/Yaskawa-Global/motoros2/pull/172))
```
Comment (gavanderhoorn) [inline CHANGELOG.md:24]:
```suggestion
- Corrected unit conversion and parameter retrieval for Base-axis TF transforms ([#172](https://github.com/Yaskawa-Global/motoros2/pull/172))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,23 @@ SPDX-License-Identifier: CC-BY-SA-4.0
...
+- Trajectories that get slowed by an FSU speed limit will now complete execution as intended ([#157](https://github.com/Yaskawa-Global/motoros2/pull/157))
```
Comment (gavanderhoorn) [inline CHANGELOG.md:25]:
```suggestion
- Improved support for trajectory execution with an active FSU ([#157](https://github.com/Yaskawa-Global/motoros2/pull/157))
```

**Pull request merged**